### ISSUE TITLE
Objective-C issues (OCLint) not reported

### ISSUE THREAD

**Issue by zippy1978** [insideapp-oss/sonar-apple#29]
### New Issue Checklist

- [x ] Updated the plugin to the latest version 
- [ x] I searched for existing [GitHub issues](https://github.com/insideapp-oss/sonar-apple/issues)

### Issue Description
When running the analysis on a project with Swift and Objective-C code, Objective-C issues are not reported.

##### Command executed
```bash
# Run tests with xcpretty to generate test report in build/reports/junit.xml
# Also saves Xcode log to build/xcodebuild.log (this is necessary for Objective-C code analysis)
# Don't forget to add -workspace to the build command if your project is part of a workspace
$ xcodebuild \
  -project MyApp.xcodeproj \
  -scheme MyApp \
  -sdk iphonesimulator \
  -destination 'platform=iOS Simulator,name=iPhone 11 Pro' \
  -derivedDataPath ./derivedData \
   clean test | tee build/xcodebuild.log | xcpretty --report junit

# Generate coverage report to build/reports/cobertura.xml
# Don't forget to activate 'Gather coverage' option in the app scheme
$ slather coverage --cobertura-xml --output-directory build/reports --scheme MyApp MyApp.xcodeproj

# Saves Periphery log to build/periphery.log (this is necessary for Swift dead code analysis)
# Don't forget to add --workspace to the build command if your project is part of a workspace
$ periphery scan \
  --project "MyApp.xcodeproj" \
  --schemes "MyApp" \
  --targets "MyApp" \
  --skip-build \
  --index-store-path ./derivedData/Index/DataStore \
  --format xcode \
  --quiet | tee build/periphery.log
  
# Run the analysis and publish to the SonarQube server
$ sonar-scanner
```

##### Complete output when running sonar-scanner, including the stack trace and command used
No OCLint issue reported into the logs.

### Environment
SonarQube Community Edition Version 9.4 (build 54424)
